queries.sql

```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN 'Jan 1, 2016' AND 'Jan 1, 2020';
+SELECT * FROM animals WHERE neutered IS TRUE AND escape_attempts < 3;
```

Comment:
- [ ] Again here we are interested in listing only the `name` COLUMN of animals that are neutured and not the full records. Kindly fix this query as well by changing `select *` to `select name`

Reply:
I will change this also thanks 😊